policy/modules/services/aptcacher.te

```diff
@@ -0,0 +1,111 @@
+policy_module(aptcacher, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+attribute_role acngtool_roles;
+
+type acngtool_t;
+type acngtool_exec_t;
+application_domain(acngtool_t, acngtool_exec_t)
+role acngtool_roles types acngtool_t;
+
+type aptcacher_t;
+type aptcacher_exec_t;
+init_daemon_domain(aptcacher_t, aptcacher_exec_t)
+
+type aptcacher_conf_t;
+files_config_file(aptcacher_conf_t)
+
+type aptcacher_var_cache_t;
+files_type(aptcacher_var_cache_t)
+
+type aptcacher_var_lib_t;
+files_type(aptcacher_var_lib_t)
+
+type aptcacher_var_log_t;
+logging_log_file(aptcacher_var_log_t)
+
+type aptcacher_runtime_t;
+files_pid_file(aptcacher_runtime_t)
```

Comment:
Nit: please we drop the "var" from the type names. I'm trying to avoid encoding paths into type names as much as possible.

Reply:
I removed all the "var" (including the var_lib ones)